pkg/cloudprovider/aws/amifamily/resolver.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/aws/karpenter/pkg/cloudprovider/aws/apis/v1alpha1"
 )
 
-var defaultEBS = v1alpha1.BlockDevice{
+var DefaultEBS = v1alpha1.BlockDevice{
```

Comment:
Does `DefaultEBS` need to be exported? It's fine if so, but I couldn't tell from the diff.

Reply:
Yeah it does.  It's used in the `aws` package.